# Message file to contain detected object position 

# detected:= object was detected
# x := horizontal distance from nose to frame center point [pixel]
# y := vertical distance from nose to frame center point [pixel]
# l := distance between left shouder and left hip or left eye to right eye [pixel]


bool detected
int16 x
int16 y
int16 l